src/BusinessTable.jsx

```diff
@@ -0,0 +1,151 @@
+            <Th>Input type status</Th>
+            <Th>Vendor type</Th>
+            <Th>Status</Th>
+            <Th>Pets of the homeless discount</Th>
+            <Th>Updated by</Th>
+            <Th>Updated date time</Th>
+            <Th>Sync to qb</Th>
+            <Th>Veterinary</Th>
+            <Th>Ressoure</Th>
+            <Th>Food</Th>
+            <Th>Donation</Th>
+            <Th>Family shelter</Th>
+            <Th>Wellness</Th>
+            <Th>Spray neuter</Th>
+            <Th>Financial</Th>
+            <Th>Re home</Th>
+            <Th>Er boarding</Th>
+            <Th>Senior</Th>
+            <Th>Cancer</Th>
+            <Th>Dog</Th>
+            <Th>Cat</Th>
+            <Th>Fph phone</Th>
+            <Th>Contact phone</Th>
+            <Th>Web notes</Th>
+            <Th>Internal notes</Th>
+            <Th>Published</Th>
+            <Th>Shelter</Th>
+            <Th>Domestic Violence</Th>
+            <Th>Web Date Init</Th>
+            <Th>Ent Qb</Th>
+            <Th>Service Request</Th>
+            <Th>Inactive</Th>
+            <Th>Final Check</Th>
+            <Th>Created By</Th>
+            <Th>Created Date</Th>
+            <Th>City</Th>
+          </Tr>
+        </Thead>
+        <Tbody>
+          {data.map((item, index) => (
+            <Tr key={index}>
+              <Td>{item.id}</Td>
+              <Td>{item.type}</Td>
+              <Td>{item.name}</Td>
+              <Td>{item.street}</Td>
+              <Td>{item.zipcode}</Td>
+              <Td>{item.state}</Td>
+              <Td>{item.qb_vendor_name}</Td>
+              <Td>{item.qb_city_state_zip}</Td>
+              <Td>{item.primary_phone}</Td>
+              <Td>{item.backup_phone}</Td>
+              <Td>{item.primary_email}</Td>
+              <Td>{item.comments}</Td>
+              <Td>{item.fax_phone}</Td>
+              <Td>{item.contact_name}</Td>
+              <Td>{item.website}</Td>
+              <Td>{item.business_hours}</Td>
+              <Td>{item.find_out}</Td>
+              <Td>{item.onboarding_status}</Td>
+              <Td>{item.join_date}</Td>
+              <Td>{item.input_type_status}</Td>
+              <Td>{item.vendor_type}</Td>
+              <Td>{item.status}</Td>
+              <Td>{item.pets_of_the_homeless_discount}</Td>
+              <Td>{item.updated_by}</Td>
+              <Td>{item.updated_date_time}</Td>
+              <Td>{item.sync_to_qb}</Td>
+              <Td>{item.veterinary}</Td>
+              <Td>{item.ressoure}</Td>
+              <Td>{item.food}</Td>
+              <Td>{item.donation}</Td>
+              <Td>{item.family_shelter}</Td>
+              <Td>{item.wellness}</Td>
+              <Td>{item.spray_neuter}</Td>
+              <Td>{item.financial}</Td>
+              <Td>{item.re_home}</Td>
+              <Td>{item.er_boarding}</Td>
+              <Td>{item.senior}</Td>
+              <Td>{item.cancer}</Td>
+              <Td>{item.dog}</Td>
+              <Td>{item.cat}</Td>
```

Comment:
since these are true or false stored within the database and not a string, when this is called its not actually going to otuput the string "true" or the string "false", instead we get nothing. use a tenary operator to get it to print true or false here like so:

{item.dog ? "True" : "False"}

do this for all attributes that are true or false.

when you setup the loop for setting up all the data entires, im sure theres a nice way to check if the type is a bool and then do this accordingly 👍